ci-operator/step-registry/baremetal/lab/post/wipe/baremetal-lab-post-wipe-commands.sh

```diff
@@ -142,9 +142,11 @@ for bmhost in $(yq e -o=j -I=0 '.[]' "${SHARED_DIR}/hosts.yaml"); do
   if [ "${#name}" -eq 0 ]; then
     echo "Unable to parse an entry in the hosts.yaml file"
   fi
-  echo "Ejecting virtual media from ${name}"
-  # shellcheck disable=SC2154
-  umount_virtual_media "${bmc_address}" "${redfish_user}" "${redfish_password}"
+  if [ "${BOOT_MODE}" == "iso" ]; then
```

Comment:
We don't need this and the variable BOOT_MODE in this step, do we?

In particular, the idea is that the eject is triggered in any case, regardless of the specific profile being run. Setting the BOOT_MODE to iso by default will create confusion to devs/users as, for example, PXE-based UPI BM profiles will not set it as they do not need this variable.

Reply:
Done